builder/upcloud/step_create_server.go

```diff
@@ -15,10 +16,10 @@ type StepCreateServer struct {
 }
 
 // Run performs the actual step
-func (s *StepCreateServer) Run(state multistep.StateBag) multistep.StepAction {
+func (s *StepCreateServer) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
 	// Extract state
 	ui := state.Get("ui").(packer.Ui)
-	service := state.Get("service").(service.Service)
+	svc := state.Get("service").(service.Service)
```

Comment:
Why change these variable names?

Reply:
To differentiate service package and service instance